templates/services/merge/consensus/teku.tmpl

```diff
@@ -12,6 +12,8 @@
         condition: service_completed_successfully{{end}}{{end}}
     networks:
       - sedge
+    depends_on:
+      - monitor
```

Comment:
Do the other CLs need this the same?

Reply:
I only added it on the execution clients